met-web/src/services/widgetService/MapService/index.tsx

```diff
@@ -56,11 +61,12 @@ interface PreviewShapefileRequest {
 
 export const previewShapeFile = async (data: PreviewShapefileRequest): Promise<GeoJSON> => {
     try {
+        const url = Endpoints.Maps.SHAPEFILE_PREVIEW;
         const formdata = new FormData();
         if (data.file) {
-            formdata.append('file', data.file);
+            formdata.append('file', data.file, data.file.name);
         }
-        const response = await http.PostRequest<GeoJSON>(Endpoints.Maps.SHAPEFILE_PREVIEW, formdata);
+        const response = await http.PostRequest<GeoJSON>(url, formdata, {}, { 'Content-type': 'multipart/form-data' });
```

Comment:
Was the linter throwing errors here?

Reply:
This was a small bugfix I implemented to fix an issue with file uploading on the Map widget that turned out not to be related to the Image widget the way I thought it was. I accidentally left it in this PR; should I remove it?

Reply:
You don't need to review it but for small, unrelated fixes, the PR description is a great place to add those!